### PR TITLE
fix: handle pay for tx correctly

### PR DESCRIPTION
**Pull request by CedrikNikita**
|develop|fix|
|---|---|
|image|image|
|image|image|

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
After I approved Tx by both signers I was not able to send it: 

tx
aa

**Comment by CedrikNikita** [comment]
> After I approved Tx by both signers I was not able to send it:

This behaviour will remain till we will push the needed fix #3015

File: src/popup/components/TransactionOverview.vue [hunk 1]
```diff
@@ -100,7 +100,27 @@ export default defineComponent({
...
+            label: t('common.smartContract'),
+            address: innerTx.value.contractId,
+          },
+        };
+      }
+
+      const transactionTag = (
+        outerTxTag.value === Tag.PayingForTx
+        || outerTxTag.value === Tag.GaMetaTx
+      ) ? innerTxTag.value : outerTxTag.value;
```
Comment (peronczyk) [inline src/popup/components/TransactionOverview.vue:121]: Maybe you can use `getTxTag` utility function?

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/TransactionOverview.vue (hunk 1 above)
Comment (CedrikNikita) [reply src/popup/components/TransactionOverview.vue:121]: `innerTxTag` is a `getTxTag` of `innerTx` and `outerTxTag` is `getTxTag` of `outerTx`. And `getTxTag` function will not return us tag that is required for the overview.

**Comment by Liubov-crypto** [comment]
LGTM

**Review by peronczyk — APPROVED** [review]

**Pull request merged**